proposals/compatibility-mode.md

```diff
@@ -228,6 +263,8 @@ sampler in a shader will generate a validation error at pipeline creation time.
 
 **Justification**: OpenGL ES 3.1 says such usage has undefined behavior.
 
+Feature: `depth-texture-sample`.
```

Comment:
`texture-depth-sample` to match `texture-depth-load`
(or vice versa)

Reply:
Done (vice versa; seems to read a little better).